docs/glossary.rst

```diff
@@ -0,0 +1,68 @@
+.. currentmodule:: plasmapy
+
+********
+Glossary
+********
+
+.. glossary::
+
+   args
+      An abbreviation for `positional arguments`_.
+
+   -like
+      Used to indicate an `object` of that type or that can instantiate
+      that type.  For example, ``"He 2+"`` is :term:`particle-like`
+      because it can be used to instantiate |Particle|.
+
+   charge number
+      The charge of a particle in units of elementary charge.
+
+   keyword-only
+      An argument that must be provided along with the name of that
+      argument. If ``z`` is a keyword-only argument to ``f(z)``, then
+      it can be provided as ``f(z=2)`` but not ``f(2)``.
```

Comment:
Could use a link to the python docs on this.

Reply:
Good idea.  I linked to the terms in the Python glossary for argument and parameter, which include the descriptions for keyword-only, etc.